pkg/ocmagenthandler/ocmagenthandler_deployment.go

```diff
@@ -349,3 +351,17 @@ func deploymentConfigChanged(current, expected *appsv1.Deployment, log logr.Logg
 
 	return changed
 }
+
+// buildEnvVars build the slice of environments to set to the OCM Agent deployment
+func (o *ocmAgentHandler) buildEnvVars() []corev1.EnvVar {
+	envs := []corev1.EnvVar{}
```

Comment:
As the function returns `EnvVar` type and to avoid potential confusion should this variable be renamed to envVars?